test/test.js

```diff
@@ -189,10 +189,10 @@ describe('Operations', function () {
   })
 
   it('should match expected value for shade', () => {
-    assert(closeXYZ(chroma.shade(20, '#4fc7ff').XYZ,
-      { X: 41.69787317690973,
-        Y: 49.72911239090596,
-        Z: 102.00870969130528 }
+    assert(closeXYZ(chroma.shade(20, '#4fc700').XYZ,
+      { X: 39.303512967548,
```

Comment:
#4fc7ff was a fully saturated colour, so the v value was "100", increasing the shade by 20 would return the same value, so I toned back the initial colour's shade